test/cmd/basicresources.sh

```diff
@@ -289,16 +289,16 @@ os::cmd::expect_success 'oc run --image=openshift/hello-openshift --restart=Neve
 os::cmd::expect_success 'oc run --image=openshift/hello-openshift --generator=job/v1 --restart=Never test4'
 os::cmd::expect_success 'oc delete dc/test rc/test2 pod/test3 job/test4'
 
-os::cmd::expect_success_and_text 'oc run --dry-run foo --image=bar -o "go-template={{.kind}} {{.apiVersion}}"'                                'DeploymentConfig v1'
```

Comment:
I'd still expect `-o yaml` to work. If this is broken, do we have confidence generic printing is still working for `run`?

Reply:
> I'd still expect -o yaml to work. If this is broken, do we have confidence generic printing is still working for run?

uh, this is `-o=go-template`, not `-oyaml`

Reply:
> I'd still expect -o yaml to work. If this is broken, do we have confidence generic printing is still working for run?

and switching to -oyaml doesn't produce meaningful results since the grep line matching logic stuff means "this version anywhere and that kind anywhere"

Reply:
I'll take `-o name` that verifies we are outputting the right types (deploymentconfig, not deployment)